docs/global-configs/external-db.md

```diff
@@ -18,6 +18,9 @@ conda install psycopg2
 
 To configure RDBMS connection, we need to edit `conf_global.yml` in `/conf` directory.
 
+!!! Note
```

Comment:
I suggest changing all references to `conf_global,yml` in the docs to `conf_client.yml` since we should assume that users are looking at the docs for the first time. Then, change this note to say that in past versions of Hummingbot, it's called `conf_global.yml`